xt_imager.py

```diff
@@ -90,7 +89,11 @@ def main():
 
 class PYTFTPServer(object):
     def __init__(self, folder):
-        self.tftp_server = tftpy.TftpServer(folder)
+        try:
+            import tftpy
+            self.tftp_server = tftpy.TftpServer(folder)
+        except ImportError:
+            log.error("You need to install tftpy: 'pip3 install tftpy'")
```

Comment:
Also you need to exit early, because right now constructor will complete as normal and the code will fail latter.